### DISCUSSION TITLE
Orario di uscita/arrivo a casa

### DISCUSSION THREAD

**Discussion by liog22** [indomus/forum#2326] [GENERALE - Discussioni generiche]
Siccome ho integrato un galaxy s22 ad HA, e se vado sul registro mi rileva quando sono entrato o uscito da una determinata zona già definita in precedenza (esempio casa o università), come faccio a creare una nuova scheda in home che mi dica l'orario di arrivo e di uscita da quei due luoghi?

**Comment by liog22** [comment]
Nessuno?

**Reply by 50494554524F** [reply]
per sapere quando lo stato un sensore è cambiato devi usare "last_updated" o "last_changed"

**Reply by liog22** [reply]
> per sapere quando lo stato un sensore è cambiato devi usare "last_updated" o "last_changed"

Riusciresti a scrivermi il codice?
